src/cron/updateSourcePublicThreshold.ts

```diff
@@ -0,0 +1,25 @@
+import { JsonContains, MoreThan, Not } from 'typeorm';
+import { Cron } from './cron';
+import { Source } from '../entity';
+import { updateFlagsStatement } from '../common';
+
+export const updateSourcePublicThreshold: Cron = {
+  name: 'update-source-public-threshold',
+  handler: async (con, logger) => {
+    logger.info('updating public threshold value for valid sources...');
+    const result = await con.getRepository(Source).update(
+      {
+        flags: JsonContains({
```

Comment:
TIL

Reply:
Unfortunately, it doesn't work with `MoreThan`, `LessThan`, etc. 😞